src/css/skills.scss

```diff
@@ -0,0 +1,130 @@
+@import './core/constants';
+@import './core/mixins';
+
+#skills {
+  padding: 35px 70px;
+  min-height: 600px;
+  background-image: url('https://static.wixstatic.com/media/05e3dc_cc646f530d6846828d53dd3216ec8023~mv2.jpg');
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: center;
+
+  .content {
+    width: 950px;
+    margin-top: 100px;
+    h2 {
+      color: $white;
+      font-size: $font-xl;
+    }
+    hr {
```

Comment:
styles based on parent is not very re-usable. Maybe just use a css class to assign to the `hr`.